geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/AccumuloVersion.scala

```diff
@@ -66,12 +66,12 @@ object AccumuloVersion extends Enumeration {
     }
   }
 
-  def ensureTableExists(connector: Connector, table: String): Unit = {
+  def ensureTableExists(connector: Connector, table: String, versioning: Boolean = true): Unit = {
```

Comment:
Not being used. I initially turned off versioning on the stats table and made this change to support that. I later decided to keep versioning, but figured I'd leave the option in the method here.